.exguard.exs

```diff
@@ -9,6 +9,12 @@ guard("credo", run_on_start: true)
 |> ignore(deps)
 |> notification(:auto)
 
+guard("mix format", run_on_start: true)
+|> command("mix format --check-formatted")
```

Comment:
This is good - letting us know it's not formatted on save, but not actually formatting on save. I like it!